doc/tutorials.rst

```diff
@@ -0,0 +1,85 @@
+ .. role:: html(raw)
+   :format: html
+
+.. _New_Users:
+
+Tutorials
+=========
+
+These step-by-step tutorials provided a gentle introduction to the basics of PennyLane. Once you have
+worked your way through the QuickStart tutorials, you will be ready to implement your own
+interesting QML problems in PennyLane!
+
+
+:html:`<h3>Learn the PennyLane basics</h3>`
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning to rotate a qubit.
+    :figure: ../examples/figures/bloch.png
+    :description: :ref:`qubit_rotation`
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning to tune a beamsplitter.
+    :figure: ../examples/figures/gaussian_transformation.png
+    :description: :ref:`gaussian_transformation`
+
+.. customgalleryitem::
+    :tooltip: Multiple expectation values, Jacobians, and keyword arguments.
+    :description: :ref:`advanced_features`
+
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning in a multi-device quantum algorithm.
+    :figure: ../examples/figures/photon_redirection.png
+    :description: :ref:`plugins_hybrid`
+
+.. customgalleryitem::
+    :tooltip: Extend PyTorch with real quantum computing power.
+    :figure: ../examples/figures/bloch.gif
+    :description: :ref:`pytorch_noise`
+
+:html:`<div style='clear:both'></div>`
+
+
+:html:`<h3>Quantum machine learning algorithms</h3>`
```

Comment:
Should we add one or two sentences here to separate these examples from the "gentle introduction" tutorials above?

Reply:
Done. Added the following two introductory sentences.

For basics:
> The following tutorials introduce the core PennyLane concepts, including QNodes,
plugins, and devices, via simple and easy to follow quantum machine learning
examples.

For the QML algorithms section:
> Take a deeper dive into quantum machine learning, exploring cutting edge algorithms
using PennyLane and near term quantum hardware.

Reply:
Fell free to modify as needed using suggested changes.

Reply:
For basics:

> The following tutorials introduce the core PennyLane concepts, including QNodes,
> plugins, and devices, via simple and easy to follow examples.
 
 For the QML algorithms section:

> Take a deeper dive into quantum machine learning by exploring cutting edge algorithms
> using PennyLane and near term quantum hardware.

Reply:
@AroosaIjaz for future reference, you can make the suggestions directly in the code review page